src/adagrams.js

```diff
@@ -1,7 +1,137 @@
+import { worker } from "@jest/reporters/build/coverage_worker";
+
+const alphaLetters = {
+  A: 9,
+  N: 6,
+  B: 2,
```

Comment:
While I really like this choice of datastructure, I feel like this would be easier to read if the letters were in order.